src/Spago/Build.hs

```diff
@@ -82,6 +100,7 @@ build BuildOptions{..} maybePostBuild = do
               shell backendCmd empty >>= \case
                 ExitSuccess   -> pure ()
                 ExitFailure n -> die [ "Backend " <> displayShow backend <> " exited with error:" <> repr n ]
+        checkImports globs
```

Comment:
I put this check after the compilation step so that I could ignore capturing/properly displaying lex/parse errors - I could move it up with some help on how to properly render the errors (is it as simple as logging them and then failing the build with a message?)

Reply:
I think it's fine to have this check after building, so that users can take care of compilation problems before build management problems :+1: